run/src/js/Box.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+class Box extends React.Component {
+  render() {
+    if (this.props.data.isCertified) {
+      return (
+        <div className="box">
+          <div className="box__container-header">
+            <h3 className="box__container-header__title">
```

Comment:
you can have conditionals inside the jsx instead of having two if statements with seemingly the same code